src/types/profile/settings.rs

```diff
@@ -9,10 +9,13 @@ use url::Url;
 pub struct Settings {
     pub interface_language: String,
     pub streaming_server_url: Url,
+    pub player_type: u8,
```

Comment:
Not sure what's the best type to use for this, so that it's still app agnostic. It could be an int, enum or a string. For Android TV I'll have these values `0=Exo/1=libVLC/2=External`

Reply:
Could we separate those settings like `internal_player` (Default/Exo/libVLC) and `external_player` (None/mpv/vlc, ...)
It will probably needed for a feature on iOS to always start the stream in external player

Reply:
Enum and string seem like a good option, although I don't fully understand the domain yet.
What are possible values for the players and how often/how much they need to change?

Reply:
i would suggest a string.
thats how we handle other settings like language, colors and font

Reply:
> Could we separate those settings like internal_player (Default/Exo/libVLC) and external_player (None/mpv/vlc, ...)

@tymmesyde Not sure why it couldn't be a single option and include both internal player(s) and the external ones?

> i would suggest a string.

Yeah, makes sense then to use it here as well.

Reply:
Just thought that if a user is setting it to Exo on his tv, it would conflict on iOS or desktop if the user chose to use external player but it was just a suggestion

Reply:
@tymmesyde settings are local per platform, they are not synced, so it can have different values per app/platform.